pytorch_lightning/trainer/connectors/accelerator_connector.py

```diff
@@ -403,7 +403,10 @@ def select_precision_plugin(self) -> PrecisionPlugin:
         raise NotImplementedError("We only support precisions 64, 32 and 16!")
 
     def select_training_type_plugin(self) -> TrainingTypePlugin:
-        if self.use_ddp2:
+        is_plugin = self.distributed_backend.training_type_plugin is not None
+        if isinstance(self.distributed_backend, Accelerator) and is_plugin:
```

Comment:
this has to be merged into the if statement. only the Accelerator object has the training_type_plugin attribute.

```suggestion
        if isinstance(self.distributed_backend, Accelerator) and self.distributed_backend.training_type_plugin is not None:
```